rllib/evaluation/episode_v2.py

```diff
@@ -184,6 +184,7 @@ def add_init_obs(
                 "_disable_action_flattening", False
             ),
             is_policy_recurrent=policy.is_recurrent(),
+            intial_states=policy.get_initial_state(),
```

Comment:
Can you explain why this needs to be here now? Is this part of the fix as well?

Reply:
yes. the agent collector needs access to the predetermined initial state in order to add it to a sample batch. 

This information was available to the policy agent collector but not the episode agent collector for some reason